### DISCUSSION TITLE
renaming folders with tag data?

### DISCUSSION THREAD

**Discussion by CeeFiddie** [Marekkon5/onetagger#302] [General]
hi marrekon/onetagger community, 

i just stumbled on onetagger yesterday.  This is incredible work you've done here.  Can I ask...is there a way to make renaming folders work for templates?  i'd like to do a :

[%catno%] %artist% - %album% (%year%) [%source% %format% %bitrate%] scheme and after auto taggimg - i've had to go to mp3tag to achieve it.  Would be amazing if it could be done all within the app.

**Comment by Marekkon5** [comment]
Hello, you can just use `/` as a folder separator (on all platforms) in the Renamer tab.

**Comment by CeeFiddie** [comment]
Hey marekkon, I'm sorry for the late response.  Thank you so much for responding.

I will try this.  Once again - i am so grateful for your work on this software - it is an incredible piece of work, i know how much effort it would have been to make something like this!

I wonder if folder renaming should have a heading?  It seems like one of the only features of the program which is a bit cryptic - the rest of the program is incredibly clear, and doesnt require reading the help even :)
